container_workflow_tool/main.py

```diff
@@ -68,8 +68,9 @@ def from_args(cls, args) -> Any:
         """
         Creates an ImageRebuilder instance from argparse arguments.
         """
-        config = getattr(args, 'config', 'default.yaml')
-        rebuilder = ImageRebuilder(base_image=args.base, config=config)
+        config = args.config if args.config else 'default.yaml'
+        config_fn, image_set = u._split_config_path(config)
```

Comment:
I don't follow what does it mean `config_fn`? Can you please rename it to something better understandable?

Reply:
I reused the original naming, I guess `fn` meant `filename`, but I can rename to `path` which is short as well and more obvious

Reply:
Done